src/versionbits.cpp

```diff
@@ -5,15 +5,21 @@
 #include <versionbits.h>
 #include <consensus/params.h>
 
-ThresholdState AbstractThresholdConditionChecker::GetStateFor(const CBlockIndex* pindexPrev, const Consensus::Params& params, ThresholdConditionCache& cache) const
+ThresholdState ThresholdConditionChecker::GetStateFor(const CBlockIndex* pindexPrev, ThresholdConditionCache& cache) const
 {
-    int nPeriod = Period(params);
-    int nThreshold = Threshold(params);
-    int64_t nTimeStart = BeginTime(params);
-    int64_t nTimeTimeout = EndTime(params);
+    int nPeriod = m_period;
+    int nThreshold = m_dep.threshold;
+    int64_t height_start = m_dep.startheight;
+    int64_t height_timeout = m_dep.timeoutheight;
+    int64_t min_activation_height = m_dep.m_min_activation_height;
```

Comment:
`height_active_min` might be more consistent with the other variables.

Reply:
Done